examples/platforms/da15000/radio.c

```diff
@@ -364,6 +387,8 @@ void otPlatRadioClearSrcMatchShortEntries(otInstance *aInstance)
 
     uint8_t i, j;
 
+    otLogDebgPlat(sInstance, "Clear ShortAddress entries", NULL);
```

Comment:
I dont think you need to pass in NULL, this should work fine with just the two arguments.